bin/cassandra

```diff
@@ -174,6 +174,23 @@ case "`uname -s`" in
     ;;
 esac
 
+clean_heap_dump_files()
+{
+    if [ "$CASSANDRA_HEAPDUMP_CLEAN" != "" ] && \
```

Comment:
what about using CASSANDRA_HEAPDUMP_KEEP_NEWEST_N_FILES instead of  CASSANDRA_HEAPDUMP_CLEAN? It is more self-explanatory..

Reply:
done